crates/holochain_conductor_api/src/config/conductor/paths.rs

```diff
@@ -42,6 +43,33 @@ impl<'a> From<&'a Path> for DatabaseRootPath {
     }
 }
 
+/// Newtype for the compiled wasms path. Has a Default.
+#[derive(
+    Clone,
+    From,
+    Into,
+    Debug,
+    PartialEq,
+    AsRef,
+    FromStr,
+    Display,
+    serde::Serialize,
+    serde::Deserialize,
+)]
+#[display(fmt = "{}", "_0.display()")]
+pub struct CompiledWasmsRootPath(PathBuf);
```

Comment:
how about a `Lazy<PathBuf>` for a default, e.g.

```suggestion
pub static DEFAULT_COMPILED_WASMS_ROOT_PATH: Lazy<PathBuf>> = Lazy::new(|| {
    data_root().join(PathBuf::from(WASM_CACHE_DIRECTORY))
}
```

this would allow us to just use vanilla `PathBuf`

Reply:
@steveej hmm but if you look at everything else in this file it follows the pattern i used, i'd prefer everything to be consistent, so if we change one we should change them all